### DISCUSSION TITLE
Using plots from a different key?

### DISCUSSION THREAD

**Discussion by berlef80** [Chia-Network/chia-blockchain#5227] [Q&A]
I have a friend who plotted some but is not interested in continuing.  I believe plots are key specific.  Is that public or private key specific?  Is there any way to transfer these plots over without using his key?  Is it possible to farm using two different keys on the same computer?
Thanks!

**Comment by lvcivs** [comment]
Plots are tied to the keys, so no it is not possible to transfer his plots to your setup. You would need to run two separate full nodes, one with his keys and one with yours. You could send the rewards to the same wallet, though.

**Reply by berlef80** [reply]
Thanks for the fast reply.  One other question.  Just curious to understand the key system better,  If someone else plots with my keys can they do that with just the public key or do they need the private key as well?  Can I generate another key and give that to them and would I then be able to transfer those plots?  Or is it really public and private key specific?  I ask because i plot on one machine with one key and then farm on another machine with another key and it works...I also have the plotting key on my farming machine so maybe it sees them as linked?

**Reply by lvcivs** [reply]
It is possible to plot for someone else using the “Farmer Public Key” (farmer_pk) and “Pool Public Key” (pool_pk). There is no need to give out any private keys.
So for example if you use a plotting service like [chiaplotting.services](chiaplotting.servies) you will need to give them your farmer public key and your pool public key and they will use those to make the plots for you. Those plots will be only usable by you.

**Reply by berlef80** [reply]
Got that part - but why can I plot on one computer using one key (different key when I enter Chia GUI and different farmer key) and then transfer those plots onto a HDD and plug it into my farming computer which has a different key when I ether Chia GUI and a different farmer key)?  I do have the other key from my plotter as an alternate key on my farming laptop GUI.  Are they linked because of that?  I did this because someone else told me to have two sperate keys - one for farming and one for plotting.

**Reply by lvcivs** [reply]
I'm surprised that that works. Yeah, maybe because you added both sets of keys on the farming machine? Or did you specify the farmer and pool keys with `-f` and `-p` when plotting? 

I don't really see an advantage in having different keys on the plotter and farmer (except maybe that if your plotting machine is hacked, your farmer will not be affected.)

**Reply by berlef80** [reply]
No im plotting with the GUI and it assigned the keys.  I just checked and they are different on the two GUI interfaces

**Reply by berlef80** [reply]
It was suggested to me here: https://github.com/Chia-Network/chia-blockchain/discussions/1116

**Reply by berlef80** [reply]
Reading that it must have to do with the private vs public key" When making a plot you can either give it your private keys (this is the default behaviour), or your public keys. Public keys are derived from the private keys but it is public information which you can share with anyone.

Plotting with public keys can't be done in the GUI right now, only the command line. In theory you can give someone your public keys, they can plot, and then they can send you the file. Farming it requires the private key."

So if I gave someone both my public keys would they plot to the same private key?  I assume that is what it is saying and why it works on my set up?
